pkg/reconciler/knativeeventing/common/images.go

```diff
@@ -33,6 +33,7 @@ import (
 var (
 	// The string to be replaced by the container name
 	containerNameVariable = "${NAME}"
+	delimiter             = "-"
```

Comment:
I thought the decision from chat was to use / as a delimiter, am I misremembering?

In particular, because - is allowed in deployment's container names, the override for

deployment-container

is ambiguous, it could mean container in deployment, or it could mean a container called deployment-container.

Reply:
It looks like to I need to switch to `/`.